### DISCUSSION TITLE
OS window decorations

### DISCUSSION THREAD

**Discussion by StevePRGM** [WerWolv/ImHex#1955] [Feedback]
I'm using Win10.

When I change settings to use OS window decorations,
 , press F11 for "full screen mode"
 , press F11 again to go back.

I get the result: loss of OS window decorations

**Comment by WerWolv** [comment]
Hey, I can't seem to reproduce this on Windows 11. Non the less, I implemented a fix now that forces the expected window decoration state back after exiting fullscreen mode. Please try build c853349b78529d102bafe4e9c91230af00260fdb and see if it still happens for you

**Comment by StevePRGM** [comment]
Yes, the 1.36.2 build fixed the issue. (I use the portable version)
